.buildkite/pipeline.build_core.yml

```diff
@@ -4,10 +4,8 @@
   instance_size: medium
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
-    - DL=1 ./ci/env/install-dependencies.sh
```

Comment:
where do we run the `test_tensorflow` now?

Reply:
it remains in the existing civ1 pipeline for now